docs/reference/licenses.md

```diff
@@ -0,0 +1,15 @@
+---
+id: licenses
+title: "Licenses"
+---
+### Licensing
+
+This page contains licensing information for all components of Camunda Cloud.
+
+#### Zeebe
+
+Licenses and license information for Zeebe can be found on the Zeebe project [README](https://github.com/camunda-cloud/zeebe#license).
+
+#### Camunda Cloud Documentation
```

Comment:
does it make sense to link here too? https://camunda.com/legal/terms/cloud-terms-and-conditions/ ?

Reply:
Good idea, I will add 🙂